src/fromager/context.py

```diff
@@ -57,6 +57,9 @@ def __init__(
         self.variant = variant
         self.jobs = jobs
         self.network_isolation = network_isolation
+        self.resolver_cache: dict[str, list[candidate.Candidate]] = (
```

Comment:
It's going to be a dict of normalized names to candidates, right? Do we have a special type for normalized names?